### PR TITLE
Fix/html css cleanup

### PR DESCRIPTION
**Pull request by elsafluss**
* Is this a fix or a feature?
fix
* What is the change/fix?
Clean up CSS and HTML to display data better and make it all more responsive
* Where should the reviewer start?
index.html and styles.css
* How should this be tested?
open index.html
* Screenshots(if appropriate)
* 
image
image

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,159 +1,169 @@
...
-            <p class="stats-text">sleep</p>
-          </div>
-        </section>
-        <section class="hydration current-stats">
-          <div class="daily-stat-icons">
-            <img src="../assets/water.svg" class="water-icon icon">
-            <p class="stats-text">water</p>
+      <div class="chooser">
+        <label for="date">Choose a start date</label>
+        <input type="date" class="date-picker" id="date" value="2019-06-15" min="2019-06-15" max="2019-09-22">
```
Comment (eddiezane) [inline src/index.html:19]: Are these min and max values intended?

File: src/index.html [hunk 2]
```diff
@@ -1,159 +1,169 @@
...
-          <p class="stairs-compare"></p>  
-          <p class="steps-compare"></p>
-          <p class="minutes-compare"></p>
-          <table class="activity-data table">
+          <div class="sleep-stats reset hidden">
+            <h2>Today's Sleep</h2>
+            <p class="today-sleep"></p>
+            <p class="avg-sleep-stats"></p>
+            <table class="sleep-data table">
+              <th class="weekly-sleep">Start date</th>
```
Comment (eddiezane) [inline src/index.html]: Do these need to be in a `tr`?

File: src/index.html [hunk 3]
```diff
@@ -1,159 +1,169 @@
...
-        </div>
-        <div class="hydration-stats reset hidden">
-          <h2>Today's Water Consumption</h2>
-          <p class="today-consumption"></p>
-          <table class="water-data table">
             <tr>
+              <th class="weekly-sleep">6 days later</th>
+              <td class="sleep six-day-sleep"></td>
+            </tr>
+              </table>
```
Comment (eddiezane) [inline src/index.html]: nit: some misaligned tags in here.

File: src/index.html [hunk 4]
```diff
@@ -1,159 +1,169 @@
...
+          </div>
+          <div class="hydration-stats reset hidden">
+            <h2>Today's Water Consumption</h2>
+            <p class="today-consumption"></p>
+            <table class="water-data table">
+              <th class="weekly-water">Start date</th>
```
Comment (eddiezane) [inline src/index.html]: If the above mentioned comments needs a `tr`, this does as well.

File: src/scripts.js [hunk 5]
```diff
@@ -212,9 +218,9 @@ function getHydrationData(placement, index) {
 function getStepData(placement, index) {
   placement.innerText = `${
     currentUser.userActivity.getWeekOfData(chosenDate, 'numSteps')[index]
-  } steps, ${
+  } steps, \n ${
```
Comment (eddiezane) [inline src/scripts.js]: Is the `\n` necessary? I believe template strings preserve lines. Same for below.

File: src/userRepo.js [hunk 6]
```diff
@@ -26,23 +26,16 @@ class UserRepo {
...
   }
 
-  // could dry up the three functions below. pass in parameter (fullList). take in 2 arguments (water, ID) fullList and ID
-  // low hanging fruit
-
   filterHydrationData(id) {
-    return this.hydrationData.filter(
-      (hydrationItem) => hydrationItem.userID === id
-    );
+    return this.hydrationData.filter((hydrationItem) => hydrationItem.userID === id);
```
Comment (eddiezane) [inline src/userRepo.js]: nit: Is styling here intentional? If a fat arrow function accepts 1 arg, the surrounding parenthesis can be omitted.

File: src/userRepo.js [hunk 7]
```diff
@@ -60,28 +53,28 @@ class UserRepo {
   }
 
   findGoodSleepers(startDate) {
```
Comment (eddiezane) [inline src/userRepo.js:55]: I'm not sure what this function is doing. What's the logic here?

**Review by eddiezane — COMMENTED** [review]
Looking good! Couple comments.

File: src/index.html (hunk 1 above)
Comment (elsafluss) [reply src/index.html:19]: Yes, we only have data for between those dates.

Comment (elsafluss) [reply src/index.html:19]: In the real world, you'd be able to select dates starting with the first day you used the app, up until the current day.

File: src/index.html (hunk 2 above)
Comment (elsafluss) [reply src/index.html]: Added a tr around all these first rows in the tables

File: src/index.html (hunk 3 above)
Comment (elsafluss) [reply src/index.html]: moved them over - also added a closing table tag on 161

File: src/index.html (hunk 4 above)
Comment (elsafluss) [reply src/index.html]: added

File: src/scripts.js (hunk 5 above)
Comment (elsafluss) [reply src/scripts.js]: Oh hey you're right! Learn something new every day!

File: src/userRepo.js (hunk 6 above)
Comment (elsafluss) [reply src/userRepo.js]: Extra () removed throughout file

File: src/userRepo.js (hunk 7 above)
Comment (elsafluss) [reply src/userRepo.js:55]: We were told to find the people whose sleepQuality averaged above a 3 for a given week.

**Comment by elsafluss** [comment]
Changes mentioned above are pushed.

**Pull request merged**